extensions/ipynb/src/ipynbMain.ts

```diff
@@ -86,8 +87,8 @@ export function activate(context: vscode.ExtensionContext, serializer: vscode.No
 		data.metadata = {
 			cells: [],
 			metadata: {},
-			nbformat: 4,
-			nbformat_minor: 2
```

Comment:
having a mismatch will cause some major issues, so I still need to investigate why that happens

Reply:
never mind, issue still exists in the ipynb extension